R/Package.R

```diff
@@ -0,0 +1,24 @@
+# @file Package.R
```

Comment:
In all other OHDSI packages this file has the name of the package, in this case 'ROhdsiWebApi.R'.